src/components/SearchBox.vue

```diff
@@ -5,111 +5,30 @@
       :isSearchStalled="state.isSearchStalled"
       :refine="state.refine"
     >
-      <form
-        action=""
-        role="search"
-        novalidate
-        :class="suit('form')"
-        @submit.prevent="onFormSubmit"
-        @reset.prevent="onFormReset"
+      <ais-search-input
+        :refine="state.refine"
+        :placeholder="placeholder"
+        :autofocus="autofocus"
+        :show-loading-indicator="showLoadingIndicator"
+        :should-show-loading-indicator="state.isSearchStalled"
+        :submit-title="submitTitle"
+        :clear-title="clearTitle"
       >
-        <input
```

Comment:
todo: add slots here to allow them to be overridden from parent components (same in RefinementList)